Makefile

```diff
@@ -25,12 +25,12 @@ clean:
 	find ./use-cases -type f -name '*.md' -not -name 'README.md' -print0 | xargs -0 rm --
 
 readme:
-	cat docs/intro.md > README.md
+	cat docs/1-intro.md > README.md
```

Comment:
What is the number for? Is it for sorting? If I decide to move about on top of implemented do I have to rename the files? If yes, does not make sense to me.

Reply:
Yes. I was just trying to make it slightly faster to go edit the right file right away.

If this makes your skin crawl, I'll just roll this back LOL. No big deal :-)

Reply:
It does, reminds me of people trying to organise word document and excel sheets 😆

Reply:
@ruflin  Sounds good.  Happy to do another review of all recent changes. Please ping me when you're ready for me to take a look.

Reply:
Here's how I organize my ECS excel sheets 😆 

```
C:\> dir
ecs.zip
ecs-backup.zip
ecs-final.zip
ecs-final-(bobs-corrections).zip
ecs-true-final.zip
ecs-last-minute-corrections-really-final.zip
```